signer/src/schnorr.rs

```diff
@@ -135,8 +139,8 @@ impl<H: 'static + Hashable> Schnorr<H> {
     fn message_hash(&mut self, pub_key: &PubKey, rx: BaseField, input: &H) -> ScalarField {
         let schnorr_input = Message::<H> {
             input: input.clone(),
-            pub_key_x: pub_key.into_point().x,
```

Comment:
kp.public is implicitly copied twice before, now only x and y are implicitly copied once respectively